packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -63,8 +141,7 @@ export const outOfOfficeCreateOrUpdate = async ({ ctx, input }: TBookingRedirect
             team: {
               members: {
                 some: {
-                  userId: ctx.user.id,
-                  accepted: true,
```

Comment:
why was `accepted: true `removed?

Reply:
when i started to develop, i remember i found a valid reason for this and felt its not necessary that's why i had removed, forgot to note down the exact reason.
May be did i think that already in listMembers.handler.ts we are doing, and result from this only we are showing in selection.
 https://github.com/calcom/cal.com/blob/6dd8812a19635456f7e8168aa480e2f16d5eb687/packages/trpc/server/routers/viewer/teams/listMembers.handler.ts#L34
 
Added back, if i recollect a strong reason for removing this, will update. Anyway it should not break in most of the cases.